Add some terms related to styles to the glossary (#38076)
WordPress/gutenberg: docs/explanations/glossary.md

The abstract term used to describe units of markup that, composed together, form the content or layout of a webpage. The idea combines concepts of what in WordPress today we achieve with shortcodes, custom HTML, and embed discovery into a single consistent API and user experience.

## Block Styles

The CSS styles that are part of the block, either via its stylesheet or via the block markup itself. For example, a class attached to the block markup is considered block styles.

> Compare to <a href="#global-styles">Global Styles</a>. In contraposition to "Global Styles", block styles are sometimes referred to as "Local Styles".

Reviewer: > Local Styles

Would it be useful to add an entry in this very glossary like so:

```markdown
## Local Styles

See <a href="#block-styles">Block Styles</a>.
```

Author: Oh, missed this comment, sorry. https://github.com/WordPress/gutenberg/pull/38083

## Block Supports

> An API for blocks to declare what feautures they support. By declaring support for a feature, the API would add aditional <a href="#attributes">attributes</a> to the block and matching UI controls for most of the existing block supports.

Reviewer 2: ```suggestion
An API for blocks to declare what features they support. By declaring support for a feature, the API would add additional <a href="#attributes">attributes</a> to the block and matching UI controls for most of the existing block supports.
```

This refers to a collection of features that ultimately allows users to edit their entire website using blocks as the starting point. This feature set includes everything from block patterns to global styles to templates to design tools for blocks (and more). First released in WordPress 5.9.

## Global Styles

> The CSS styles generated by WordPress and enqueued as a embedded stylesheet in the front end of the site. The stylesheet ID is `global-styles-inline-css`. The contents of this stylesheet come from the default `theme.json` of WordPresss, the `theme.json` theme, and the styles provided by the user via the global styles sidebar in the site editor.

Reviewer 3: ```suggestion
The CSS styles generated by WordPress and enqueued as an embedded stylesheet in the front end of the site. The stylesheet ID is `global-styles-inline-css`. The contents of this stylesheet come from the default `theme.json` of WordPress, the theme's `theme.json`, and the styles provided by the user via the global styles sidebar in the site editor.
```

> The CSS styles generated by WordPress and enqueued as a embedded stylesheet in the front end of the site. The stylesheet ID is `global-styles-inline-css`. The contents of this stylesheet come from the default `theme.json` of WordPresss, the `theme.json` theme, and the styles provided by the user via the global styles sidebar in the site editor.

Reviewer 2: ```suggestion
The CSS styles generated by WordPress and enqueued as a embedded stylesheet in the front end of the site. The stylesheet ID is `global-styles-inline-css`. The contents of this stylesheet come from the default `theme.json` of WordPress, the `theme.json` theme, and the styles provided by the user via the global styles sidebar in the site editor.
```
